### DISCUSSION TITLE
Restrict testcase size

### DISCUSSION THREAD

**Discussion by antonmedv** [google/oss-fuzz#11190] [Q&A] [answered]
Is there a way to restrict testcase sizes? 

My parser crashes on very big inputs.

**Comment by DavidKorczynski** [comment, marked as answer]
A common way is having a guard in the harness itself, e.g.

https://github.com/google/oss-fuzz/blob/ab2a4151bd03b67f2d3e9a2418838447f43067e5/projects/zlib/compress_fuzzer.c#L63-L64

**Reply by antonmedv** [reply]
Got it. Thanks.
